src/cpu/meson.build

```diff
@@ -14,32 +14,34 @@ conf_data.set10('C_PER_PAGE_W_OR_X', per_page_w_or_x_pref.auto() or per_page_w_o
 
 # Platform-specific
 core_selection = [
-    # cpu_family                  selected_core        dynrec_define    target     unaligned  per-page
-    #                                                                              mem        W^X
-    [ ['x86_64'],                 ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86_64',  1,         1 ],
-    [ ['x86'],                    ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86',     1,         1 ],
-    [ ['x86_64'],                 ['dynrec'],          'C_DYNREC',      'X86_64',  1,         1 ],
-    [ ['x86'],                    ['dynrec'],          'C_DYNREC',      'X86',     1,         1 ],
-    [ ['aarch64'],                ['auto', 'dynrec'],  'C_DYNREC',      'ARMV8LE', 1,         1 ], # ARMv8+ (64-bit)
-    [ ['arm'],                    ['auto', 'dynrec'],  'C_DYNREC',      'ARMV7LE', 1,         1 ], # ARMv7+
-    [ ['armv6'],                  ['auto', 'dynrec'],  'C_DYNREC',      'ARMV4LE', 0,         0 ], # ARMv6 or older
-    [ ['ppc64',   'powerpc64'],   ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         1 ], # 64 bit PPC processors (big-endian), running 32-bit big-endian code
-    [ ['ppc64le', 'powerpc64le'], ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors (little-endian)
-    [ ['ppc',     'powerpc'],     ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         0 ], # 32 bit PPC processors (big-endian)
-    [ ['mips'],                   ['auto', 'dynrec'],  'C_DYNREC',      'MIPSEL',  0,         0 ], # 32 bit MIPS processor
+    # cpu_family              endian      selected_core        dynrec_define    target     unaligned  per-page
+    #                                                                                      mem        W^X
+    [ ['x86_64'],             ['little'], ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86_64',  1,         1 ],
+    [ ['x86'],                ['little'], ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86',     1,         1 ],
+    [ ['x86_64'],             ['little'], ['dynrec'],          'C_DYNREC',      'X86_64',  1,         1 ],
+    [ ['x86'],                ['little'], ['dynrec'],          'C_DYNREC',      'X86',     1,         1 ],
+    [ ['aarch64'],            ['little'], ['auto', 'dynrec'],  'C_DYNREC',      'ARMV8LE', 1,         1 ], # ARMv8+ (64-bit)
+    [ ['arm'],                ['little'], ['auto', 'dynrec'],  'C_DYNREC',      'ARMV7LE', 1,         1 ], # ARMv7+
+    [ ['armv6'],              ['little'], ['auto', 'dynrec'],  'C_DYNREC',      'ARMV4LE', 0,         0 ], # ARMv6 or older
+    [ ['ppc64', 'powerpc64'], ['big'],    ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         1 ], # 64 bit PPC processors (big-endian), running 32-bit big-endian code
```

Comment:
```suggestion
    [ ['ppc64', 'powerpc64'], ['big'],    ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         1 ], # 64 bit PPC processors (big-endian), running 64-bit big-endian code
```

Reply:
I also found that comment surprising. But, my attempt here is to avoid changing anything for any other row that the 64-bit little-endian PowerPC one, so I refrained from making any other changes than adding the `endian` column.

I can add a comment change if it is certain that the old comment was wrong. However, I suspect it is actually correct. The failure that requires this fix is that ppc64le arch selected the highlighted row, and build failed because the its code path contains pointer conversions to 32-bit integers. To me, that sounds like "running 32-bit code" is actually accurate. (Talking as somebody who knows very little about PowerPC, and has just general understanding of ISAs.)